### PR TITLE
Cross sell card click listener

### PR DESCRIPTION
**Pull request by hugokallstrom**
### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

**Review by StylianosGakis — APPROVED** [review]

File: app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt [hunk 1]
```diff
@@ -70,7 +71,11 @@ fun CrossSell(
             ),
             contentDescription = null,
             contentScale = ContentScale.Crop,
-            modifier = Modifier.fillMaxSize(),
+            modifier = Modifier
+                .fillMaxSize()
+                .clickable(
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt:76]: Does this make the card have a ripple? 🙂

**Review by hedvigoscar — APPROVED** [review]
Approved with nits

File: app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt (hunk 1 above)
Comment (StylianosGakis) [reply app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt:76]: There is a different overload of the `Card` composable that does have an `onClick`, but it is currently annotated with `@ExperimentalMaterialApi` (not sure why, maybe they're likely to change its API).
So if this one doesn't behave as expected maybe that one is the solution.

Comment (hugokallstrom) [reply app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt:76]: It does make a ripple! :)

**Pull request merged**